### ISSUE TITLE
Discussion on Readme Semantics 

### ISSUE THREAD

**Issue by rodsouza9** [Pantherbotics/Club-Media#1]
> DO NOT INSTALL THIS ON YOUR PERSONAL COMPUTER - without permision.
> One entire factor of git hub is ope source. This should be phrased differently. The proceeding lines appear to be a warning to this should be something like 
> 
> **WARNING: Attempting to clone this repo may take hours** due to the large pictures...

**Comment by rogerfachini** [comment]
@rodsouza9 Github is not _only_ about open-source projects. You can have different licenses for different projects. Look it up sometime if you can ever be bothered to. In addition, this is a test page, and per the discussions Sam and I had on Saturday, it seems like not the most viable option for storing our media, and is very much in an alpha state. Therefore, it is not something in need of nitpicking of semantics details.

**Comment by rodsouza9** [comment]
I am aware that it isn't only open source. But since this repo is open source i just thought the "do not install" thing would be reworded. I appreciate you trying to point out these small things, but for the sake of being concise, lets try to keep the comment as close to the topic at hand.

**Comment by rogerfachini** [comment]
Yes, as far as sticking to the topic at hand, here are a few things that may help. Often users will interpret warnings merely as suggestions, and Sam doesn't want this installed on anyone's computer unless they are actively editing club media, presuming our WIP system is viable at all. The whole point is that only people who need access to the media will want it cloned, and for good reason too. We want to prevent people from cloning it unless they need to, hence the strongly worded readme.

**Comment by rodsouza9** [comment]
I see where you are coming from, but as a public repository, any one that is capable of using GitHub, is allowed to clone this repository. If we really do not want any one to clone this repository. Then we make this repository private. But I do not believe that is necessary or worth the cost. So instead of out right telling people you are not allowed to do something, give them a warning. People are more generally going to listen to a **friendly warning** rather than **straight up telling them that they are not allowed to do something**... just saying.

**Comment by RoboticRice** [comment]
I have made some modifications to the wording. I don't really give a damn either way, but I'll usually err on the side of caution, so I changed to wording. You have my permission to change the formatting (b/c I suck at git formatting) if you would like, but please do NOT change the wording without asking first.
